src/app/delegation_backend/src/delegation_backend/constants.go

```diff
@@ -18,7 +18,7 @@ var SIG_PREFIX = [...]byte{1}
 var BLOCK_HASH_PREFIX = [...]byte{1}
 
 func NetworkId() uint8 {
-	if os.Getenv("NETWORK") == "" {
```

Comment:
Is this change intentional? Looks a bit out-of-context in this PR

Reply:
This is intentional, because this variable is tied together with the MF deployment procedure.